src/api/schemas/business-schema.ts

```diff
@@ -52,7 +57,8 @@ const createBusiness = object({
         business: object({
             businessId: businessId,
             name: name,
-            address: address
+            address: address,
```

Comment:
For now, I guess that the address as a field is correct but we need to talk about it because the library that we are using doesn't show how to make it a single input. I have it down to filling the corresponding field that were already present but I'll investigate further if we really want a single field.

Reply:
Just realize it's not really related to this PR, but it's still a valid question to investigate.